sparse_strips/vello_api/src/paint.rs

```diff
@@ -127,6 +112,11 @@ impl From<Gradient> for PaintType {
 
 impl From<Image> for PaintType {
     fn from(value: Image) -> Self {
+        assert!(
+            value.width < u16::MAX as u32 && value.height < u16::MAX as u32,
+            "the image is too big, the width and height can be no larger than {} pixels",
+            u16::MAX,
+        );
         Self::Image(value)
     }
 }
```

Comment:
How do we want to deal with fallible paint types?

Reply:
(Note in this case we could instead just forward the width and height as `u32` into the `EncodedImage`, but currently we convert the width and height to `u16` similarly to the other dimensions in the pipeline.)

Reply:
I think panicking is probably fine for now? u16::MAX is pretty large for a bitmap!